default-run.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+mkdir -p target/website
+mkdir -p target/transactions
+
+# The port of the underlying HTTP server
+export PORT=8084
```

Comment:
is it worth making this `export PORT="${PORT:-8084}"` so it can be overridden from the environment?

Reply:
I don't think that will work locally as at least 1 other app does that should you would get incorrect config for one of the apps.